gui/tasks/forms.py

```diff
@@ -186,6 +187,8 @@ def check_rpath_exists(self):
         exists or not. Returns TRUE rpath is a directory
         and exists, else FALSE"""
 
+        self.ssh_host_keyscan()
```

Comment:
Then this approach is wrong, you can use ssh command line options to ignore host key.